commit 788a26209d187f5edf218e61e94ff33121b05a24 (refs/changes/75/1275/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-01-19T01:21:47+00:00 (2 years, 9 months ago)
    
    MORAY-388 Moray needs "microtime" dependency back
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index e3d6eae..5477297 100644
--- a/package.json
+++ b/package.json
@@ -21,6 +21,7 @@
         "node-manatee": "git+https://github.com/joyent/node-manatee.git#152d3e1",
         "manatee": "git+https://github.com/joyent/manatee.git#8615736",
         "moray": "git+https://github.com/joyent/node-moray.git#fd5781b",
+        "microtime": "0.5.1",
         "once": "1.3.0",
         "panic": "0.2.1",
         "pg": "2.11.0",
